static/js/settings_notifications.js

```diff
@@ -203,7 +209,29 @@ export function set_up(settings_panel) {
     });
 
     set_enable_marketing_emails_visibility();
+
     rerender_ui();
+
+    // Has to be after rerender_ui because relevant rows in table are added there
+    container.find(".unmute_stream").on("click", (e) => {
+        const row = $(e.currentTarget).parent().parent();
+        const stream_id = Number.parseInt(row.attr("data-stream-id"), 10);
+        const sub = sub_store.get(stream_id);
+
+        stream_settings_ui.set_muted(sub, !sub.is_muted);
+
+        $(row).toggleClass("control-label-disabled", false);
+        $(row).find(".fa-bell-slash-o").hide();
+        $(row).find(".disabled_checkbox").attr("disabled", false);
+
+        const setting_name = $(e.currentTarget).attr("name");
+        const setting_value = settings_org.get_input_element_value(this);
+        change_notification_setting(
+            setting_name,
+            setting_value,
+            row.closest(".subsection-parent").find(".alert-notification"),
```

Comment:
I cannot understand which notification setting is being changed here. There is no `name` attribute for the unmute icon element.

Reply:
@sahil839 i agree that this is not very pretty, i used this to display this green "saving...saved" box when you click on the bell and unmute the stream. 
I changed it and wrote a separate function that does this, is that better? Or if not, how should this be done instead?